web/src/containers/admin/AdminDashboard.js

```diff
@@ -0,0 +1,60 @@
+import React, { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+
+import Icon, { faUserCog, faUserPlus } from '../../components/Icons';
+import Footer from '../../components/Footer';
+import Header from '../../components/Header';
+
+const AdminDashboard = () => {
+  return (
+    <Fragment>
+      <Header />
+      <div className="site-body ds-l-container">
+        <div className="ds-l-row">
+          <div className="ds-l-col--8 ds-u-margin-x--auto ds-u-padding-y--6">
+            <h1 className="ds-h1">
```

Comment:
For some reason, the design system doesn't style plain ol' `h` tags -- you have to use the class in conjunction with the heading. We could apply the class styles to unclassed headings at some point in the future.

Reply:
Would it make sense to have a set of React components for `<H1>`, `<H2>`, etc., that applied the classes?  Then if we used those components, we'd get the classes for free everywhere, and if we wanted to change the global appearance of a header, we could do it in one place?

```js
const H1 = ({ children }) => (<h1 className="ds-h1">{ children }</h1>);
```

But that may be a layer of abstraction too far, I dunno. :shrugging-person-made-of-symbols: